plugins/cpp_metrics/test/sources/parser/CMakeLists.txt

```diff
@@ -3,5 +3,6 @@ project(CppMetricsTestProject)
 
 add_library(CppMetricsTestProject STATIC
   mccabe.cpp
+  typemccabe.cpp
```

Comment:
Maybe also rename the `mccabe.cpp` as well.

Reply:
Solved in [62406a3](https://github.com/Ericsson/CodeCompass/pull/727/commits/62406a35283dc77fc5046950a96f49f252c9dbda).